torch/_decomp/decompositions.py

```diff
@@ -1000,6 +1000,34 @@ def _fused_dropout_decomposition(input, p, generator=None):
     return (res, mask)
 
 
+@register_decomposition(aten._to_copy)
+def _to_copy(
+    x: Tensor,
+    *,
+    dtype: Optional[torch.dtype] = None,
+    layout=None,
+    device: Optional[torch.device] = None,
+    pin_memory: bool = False,
+    non_blocking: bool = False,
+    memory_format: Optional[torch.memory_format] = None,
+):
+    assert not layout or layout == torch.strided, "TODO"
+    assert not pin_memory, "TODO"
+    assert device is not None or dtype is not None
+    dtype_converted = False
+    if device is not None and device != x.get_device():
+        # avoid conversions on cpu
+        if dtype is not None and device.type == "cpu":
+            x = torch._prims.convert_element_type(x, dtype)
+            dtype_converted = True
+        x = torch._prims.device_put(x, device)
+    if dtype and not dtype_converted:
+        x = torch._prims.convert_element_type(x, dtype)
+    if memory_format:  # no ref/prim for memory format
+        x = x.to(memory_format=memory_format)
```

Comment:
I'm somewhat worried this is gonna be recursive 🤔

Reply:
OpInfo dtype conversions tests pass

Reply:
And how is this function appearing in the dtype conversions tests? Can you please give a specific command to launch a test that uses this implementation?

Reply:
`python test/test_decomp.py -v -k test_comprehensive_short_cpu_float16`

Reply:
Thank you! Now I know about `.short()` and other dtype methods OpInfos.